crates/rolldown/src/stages/generate_stage/code_splitting.rs

```diff
@@ -273,9 +272,19 @@ impl GenerateStage<'_> {
         // the wrapped module are usually lazy evaluate). So we need to adjust the initialization
         // order
         // manually.
+        let imported_symbol_owner_from_other_chunk = chunk
+          .imports_from_other_chunks
+          .iter()
+          .flat_map(|(_, import_items)| {
+            import_items
+              .iter()
+              .map(|item| self.link_output.symbol_db.canonical_ref_for(item.import_ref).owner)
+          })
+          .collect::<FxHashSet<_>>();
```

Comment:
Should also consider implicit dependency relationships for a module that is imported from another chunk.